sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedPlainValuesReader.java

```diff
@@ -61,6 +61,14 @@ public final void readBooleans(int total, WritableColumnVector c, int rowId) {
     }
   }
 
+  @Override
+  public final void skipBooleans(int total) {
+    // TODO: properly vectorize this
```

Comment:
Shall we file a JIRA and make this IDed TODO?

Reply:
This follows a few other TODOs in the file when handling booleans. Yes I can file a JIRA too

Reply:
Created https://issues.apache.org/jira/browse/SPARK-35867

Reply:
Thanks!